api/graphite.go

```diff
@@ -177,7 +284,7 @@ func (s *Server) renderMetrics(ctx *middleware.Context, request models.GraphiteR
 
```

Comment:
there is a new implication here: aggsettings must be consistent across all peers.  seems obvious, but we should document it in the config files.